plugins/AudioFileProcessor/AudioFileProcessorWaveView.cpp

```diff
@@ -376,14 +382,16 @@ void AudioFileProcessorWaveView::zoom(const bool out)
 void AudioFileProcessorWaveView::slide(int px)
 {
 	const double fact = qAbs(double(px) / width());
-	auto step = range() * fact * (px > 0 ? -1 : 1);
+	double step = range() * fact * (px > 0 ? 1 : -1);
```

Comment:
No need to switch these to `double`. This was switched to `auto` to allow the right hand side to completely determine the type of the variable, avoiding implicit conversions.

```suggestion
	auto step = range() * fact * (px > 0 ? 1 : -1);
```